### PR TITLE
Meet Marketplace requirements

### LINKED ISSUE #133: Publish sonar-perl to the Marketplace
https://community.sonarsource.com/t/deploying-to-the-marketplace/35236

### PR DESCRIPTION
**Pull request by otrosien**
3. The key of your plugin must be:
 e. not just the name of a language (e.g. cannot be java, rust, js/javascript, ...)

https://community.sonarsource.com/t/deploying-to-the-marketplace/35236

Closes #133

### CODE REVIEW

**Comment by otrosien** [comment]
To figure out: What's the impact for upgrading for existing users of sonar-perl? To my understanding you will need to uninstall the current plugin and you will lose project annalysis history.

File: sonar-perl-plugin/build.gradle [hunk 1]
```diff
@@ -37,7 +37,7 @@ jar {
         attributes "Build-Jdk": System.getProperty('java.version'),
                    "Build-Time": new Date().format("yyyy-MM-dd'T'HH:mm:ssZ"),
                    "Plugin-BuildDate": new Date().format("yyyy-MM-dd'T'HH:mm:ssZ"),
-                   'Plugin-Key': 'perl',
+                   'Plugin-Key': 'sonarperl',
```
Comment (milbrandt) [inline sonar-perl-plugin/build.gradle]: Key should not include `sonar` as they might consider to exclude that in their rules, see the discussion with Ann at
https://community.sonarsource.com/t/new-release-perl-plugin-0-4-0/5569/8

How about `communityperl`?

File: sonar-perl-plugin/src/test/java/com/github/sonarperl/critic/PerlCriticIssuesLoaderSensorTest.java [hunk 2]
```diff
@@ -56,7 +57,7 @@ public void should_not_throw_on_report_file_not_found() {
         String relativePath = "lib/Sample/Project.pm";
         inputFile(relativePath);
         context.settings().setProperty(PerlCriticProperties.PERLCRITIC_REPORT_PATH_KEY, "src/test/resources/basic/perlcritic_nonexistant_report.txt");
```
Comment (milbrandt) [inline sonar-perl-plugin/src/test/java/com/github/sonarperl/critic/PerlCriticIssuesLoaderSensorTest.java]: Complete class `Settings` is deprecated and will be removed with Sonarqube 8.

Comment (otrosien) [reply sonar-perl-plugin/src/test/java/com/github/sonarperl/critic/PerlCriticIssuesLoaderSensorTest.java]: Thanks for the reminder.

**Pull request merged**